docs/installation.rst

```diff
@@ -14,13 +14,11 @@ simple TARDIS calculations.
 .. _requirements_label:
 
 
-.. warning::
-
-    TARDIS is only compatible with Python >3.6
-
 .. note::
-    We strongly recommend installing TARDIS within an Anaconda environment and
-    to always use the latest GitHub development version.
+        
+    - TARDIS is only compatbile with Python >3.6
+    - TARDIS only supports Linux and MacOs.
+    - We strongly recommend to install TARDIS within an Anaconda environment and to always use the lastest github development version.
```

Comment:
-Latest is misspelled 
- perhaps could use installing instead of to install 
-Do you think you should use the capitalized version of GitHub?

Reply:
Thanks! Great suggestions, I fixed all of them.